agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -18,6 +18,7 @@ package credentialspec
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
Why do you need this library?

Reply:
It's same. Look at https://github.com/aws/amazon-ecs-agent/blob/feature/gMSA/agent/utils/ioutilwrapper/ioutil.go.

Reply:
The Wrapper that ECSAgent team uses doesn't have ReadFile() which is there in the original library. I was using this method earlier.  Since now i removed the ReadFile call i don't need this io util anymore. Removed